config/openhpc.yml

```diff
@@ -74,13 +74,6 @@ openhpc_slurm_partitions:
 
 # A list of OpenHPC runtime libraries to install on compute and control nodes
 openhpc_packages:
-  - strace
```

Comment:
Would it be better to have these packages as a variable under group_vars/all/alaska since there are now multiple openhpc config files that require these common packages.

Reply:
I started the process under this pull request: https://github.com/stackhpc/p3-appliances/pull/21